covid_moonshot_ml/datasets/utils.py

```diff
@@ -4,7 +4,7 @@
 import pandas
 from rdkit.Chem import CanonSmiles, FindMolChiralCenters, MolFromSmiles
 import re
-
+import csv 
```

Comment:
is there a reason to use `csv` instead of `pandas`? just for speed purposes?

Reply:
mostly just did what I'm used to - there some advantages with handling large datasets using a CSV iterator. Changed to `pandas` to align with the rest of the codebase, see https://github.com/choderalab/covid-moonshot-ml/pull/42/commits/94965a717af26883e9dd701470c6970c32744f7f